resources/ory/charts/hydra/templates/deployment.yaml

```diff
@@ -40,75 +39,40 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
     {{- end }}
-    {{- if .Values.hydra.autoMigrate }}
-      initContainers:
-        {{- if not .Values.global.ory.hydra.persistence.gcloud.enabled }} #Use only for in-cluster databases that may not be ready yet. External databases are made ready before installation.
-        - name: wait-for-db
```

Comment:
Is this OK to be deleted? The wait-for-db.sh script wasn't deleted either way

Reply:
Yes! Please do not remove wait-for-db